.github/workflows/acceptance-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Set up JDK 8
         uses: actions/setup-java@v2
         with:
-          java-version: '8'
+          java-version: '17'
```

Comment:
Update the comment line 20 if we're running the acceptance-tests against java 17